commons/src/components/MessageBody.svelte

```diff
@@ -33,11 +33,14 @@
       for (const [fileIndex, file] of message.files.entries()) {
         if (
           file.content_disposition === "attachment" &&
-          !(file.content_id && InlineImageTypes.includes(file.content_type)) && // treat all files with content_id as inline
+          !(
+            file.content_id &&
+            message.cids?.includes(file.content_id) &&
+            InlineImageTypes.includes(file.content_type)
+          ) &&
```

Comment:
This condition check can be extracted into a function for better readability.

Reply:
Good point! I will do that :)

Reply:
Done https://github.com/nylas/components/pull/359/commits/82a52d9108ba3e53ba07e49e470c2838c73fbf35